cocos/render-scene/scene/spot-light.ts

```diff
@@ -174,6 +176,19 @@ export class SpotLight extends Light {
         this._needUpdate = true;
     }
 
+    /**
+     * @en The angle attenuation strength of the spot light.
+     * The larger the value, the softer the edge, and the smaller the value, the harder the edge.
+     * @zh 聚光灯角度衰减强度。值越大，边缘越柔和，值越小，边缘越硬。
+     */
+    get spotAngleAttenuationStrength (): number {
```

Comment:
As this function is defined in Spot, i think use `get angleAttenuationStrength` is suitable.